{
    "id": "problem_58",
    "description": "\n\nThe problem is an IndexError when trying to access the index of a pandas DataFrame. When attempting to access a specific row in the DataFrame, an IndexError is thrown, indicating that the index is out of bounds.",
    "output": "Traceback (most recent call last):\n  File \"<stdin>\", line 7\n    'Occupation\n               ^\nSyntaxError: EOL while scanning string literal",
    "tags": [
        "IndexError",
        "pandas",
        "Medium"
    ]
}